assets/model_monitoring/components/src/model_performance_metrics/data_reader.py

```diff
@@ -0,0 +1,76 @@
+from shared_utilities.io_utils import read_mltable_in_spark
+
+
+class DataReaderFactory:
+
+    def __init__(self):
+        """
+
```

Comment:
Could you please add some description for all the docstring in this file? That will be very helpful for others with no knowledge in this area to pick up.